clis/daemon-common/src/coms.rs

```diff
@@ -0,0 +1,143 @@
+//! Code for handling inter process communication and abstracting away it's dependencies.
```

Comment:
file name "coms.rs" is ambiguous, if it's for communications it should be "comms.rs"/"comm.rs"